egs/wsj/s5/steps/nnet3/report/generate_plots.py

```diff
@@ -48,6 +48,7 @@ def GetArgs():
     parser.add_argument("--comparison-dir", type=str, action='append', help="other experiment directories for comparison. These will only be used for plots, not tables")
     parser.add_argument("--start-iter", type=int, help="Iteration from which plotting will start", default = 1)
     parser.add_argument("--is-chain", type=str, default = False, action = train_lib.StrToBoolAction, help="Iteration from which plotting will start")
+    parser.add_argument("--is-linear-objf", type=str, default = True, action = train_lib.StrToBoolAction, help="Nnet trained with linear objective as against with quadratic objective")
```

Comment:
Actually I think you can make this multiple choice variable with options `{linear,quadratic}`. This would help future-proof the code as we can add more cost functions.